src/common/hooks/useHoldingsAndLocations/useHoldingsAndLocations.js

```diff
@@ -1,23 +1,30 @@
 import { useQuery } from 'react-query';
 
-import {
-  batchRequest,
-  HOLDINGS_API,
-  LIMIT_MAX,
-  LOCATIONS_API,
-} from '@folio/stripes-acq-components';
+import { LIMIT_MAX } from '@folio/stripes-acq-components';
 import {
   useNamespace,
   useOkapiKy,
 } from '@folio/stripes/core';
 
+import {
+  getHoldingLocations,
+  getHoldingLocationsByTenants,
+} from '../../utils/getHoldingLocations';
+
 const DEFAULT_DATA = [];
 
-export const useHoldingsAndLocations = ({ instanceId, tenantId, options = {} }) => {
-  const {
-    enabled = true,
-    ...queryOptions
-  } = options;
+export const useHoldingsAndLocations = ({
+  instanceId,
+  options = {},
+  tenantId,
+  /*
+   `receivingTenantIds` are uniq list of tenantIds from pieces list.
+    The purpose is, we need to be able to fetch locations from other
+    tenants so that we can display all the locations in the full screen page
+  */
```

Comment:
```suggestion
  /*
   `receivingTenantIds` is a unique list of tenantIds from the pieces list.
    The purpose is, that we need to be able to fetch locations from other
    tenants so that we can display all the locations on the full-screen page
  */
```

Reply:
updated